kolibri/plugins/coach/assets/src/views/common/QuizLessonDetailsHeader.vue

```diff
@@ -72,6 +86,9 @@
       resource() {
         return this.examOrLesson === 'lesson' ? this.lesson : this.exam;
       },
+      isFromOldKolibiri() {
+        return this.exam.data_model_version < 3 && this.exam.draft && !this.exam.active;
```

Comment:
We could refactor this into several props instead for better clarity, testability and maintainability;
```
isOldDataModelVersion() {
    return this.exam.data_model_version < 3;
}

isDraft() {
    return this.exam.draft;
}

isActive() {
    return this.exam.active; 
}

isFromOldKolibiri() {
    return this.isOldDataModelVersion() && this.isDraft() && !this.isActive();
}
```

And in the alert, it would now be;
```
<UiAlert v-if="isFromOldKolibiri">
    .....
</UiAlert>
```

Reply:
This should take into consideration @nucleogenesis's feedback [here](https://github.com/learningequality/kolibri/pull/13045#discussion_r1951361049) and [here](https://github.com/learningequality/kolibri/pull/13045#discussion_r1951557805)